requests_oauthlib/oauth2_session.py

```diff
@@ -317,6 +333,7 @@ def request(self, method, url, data=None, headers=None, withhold_token=False,
         """Intercept all requests and add the OAuth 2 token if present."""
         if not is_secure_transport(url):
             raise InsecureTransportError()
+
```

Comment:
Legibility changes are fine, but they're better in seaparate PRs so that I can keep track of what is going on